src/extension.ts

```diff
@@ -39,6 +41,21 @@ class PSCompletionProvider implements CompletionItemProvider {
 }
 
 export async function activate(context: ExtensionContext) {
+  // TODO: Extract to function
+  Object.keys(snippets).map(key => {
+    snippets[key].body = parseSnippet(snippets[key].body, 'snippet', {
```

Comment:
So why don't we create function `parseSnippets` right away?

Reply:
I used the [WIP] label in the PR's title to get the earliest feedback possible on how snippet parsing is handled. Nevertheless, the function is extracted now. Could you have a look?